### DISCUSSION TITLE
Upgrading to docker-extracted binary cant find sqllite db (but we use postgres)

### DISCUSSION THREAD

**Discussion by utvikler** [dani-garcia/vaultwarden#5241] [Q&A] [answered]
Hi,
  We've been using a self-compiled 1.30.0-03 version with postgresql for some time, and would like to run server:latest via docker-extract.  We have a couple of problems:

1. The server:latest is dynamically linked with maria, but we dont have maria/mysql, only postgresql. So, we tried with statically-linked server:alpine.
2. server:alpine-latest is supposed to be compatible with postgresql, but when starting it complains that it cannot find the sqllite database, and doesnt seem to recognize that we have a postgres installation

Our current vaultwarden.conf is in /etc/vaultwarden/ and contains the correct DATABASE_URL, is there some way to tell the vaultwarden binary to use /etc/vaultwarden/vaultwarden.conf ?

**Comment by stefan0xC** [comment, marked as answer]
> is there some way to tell the vaultwarden binary to use /etc/vaultwarden/vaultwarden.conf ?

You can set the `ENV_FILE` variable to that location.
https://github.com/dani-garcia/vaultwarden/blob/da3701c0cfa5fb0fe505c18c5f210edb2a71aaf9/.env.template#L8-L11

Alternatively if you are using a systemd .service file you can set the [`EnvironmentFile=`](https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#EnvironmentFile=) directive instead.

**Comment by utvikler** [comment]
Thank you, I got it sorted.  I had not duplicated the data directory from the current installation.  I also note that the docker binary-extraction instructions do specify that an alpine version is needed (assuming static linking is desired), so another rtfm for me.  Thanks again.
